config/event_handling.tmpl.yml

```diff
@@ -317,7 +319,7 @@ event_groups:
     - VM_WAS_SET_DOWN_DUE_TO_HOST_REBOOT_OR_MANUAL_FENCE
     - VM_RENAMED
     - hardware.ipmi.metrics.update
-    - POD_NODESELECTORMISMATCHING
+    - POD_SCHEDULED
```

Comment:
I think ```POD_SCHEDULED``` should be either under ```status``` or ```addition``` (we don't have an event for pod creation and this one is the closest one)